### ISSUE TITLE
Training with high resolution images

### ISSUE THREAD

**Issue by VeeranjaneyuluToka** [open-mmlab/mmdetection#12376]
Thanks for your error report and we appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
3. The bug has not been fixed in the latest version.

**Describe the bug**
Input resolution is very important when a network is designed such that it will have the appropriate receptive field. Changing the input resolution (either decrease or increase) will lead to the change in receptive field also. If that is the case, then how come the network produces better accuracy by training with higher resolution images esp for smaller objects?

And also, when i train with higher resolution images, is it good idea to initialize the network with pre-trained weights that are trained with 640x640 or with some other resolution?

**Comment by tojimahammatov** [comment]
Hi, as you have mentioned changing the input resolution leads to the change in receptive field. That means you will have feature maps with higher spatial sizes (width and height) when input size is big. Obviously, those feature maps preserve better insights about small objects which eventually leads to better regression or classification. 

As you decrease the input size small objects become difficult to detect since now they are even more smaller and they can represent a few receptive fields only.

For the second question, I guess it is good to have the same input sizes of pretrained weights, but if there is no any such pretrained weights, still other weights trained with different resolution would be helpful rather than random initialization. It's based on my experience only, I don't have any fact though.
